pkg/services/sqlstore/sqlstore.go

```diff
@@ -89,9 +79,18 @@ func ProvideService(cfg *setting.Cfg, cacheService *localcache.CacheService, mig
 	}
 	s.tracer = tracer
 
-	s.initMetrics()
+	// initialize and register metrics wrapper around the *sql.DB
+	db := s.engine.DB().DB
+
+	var metrics prometheus.Collector
+	if cfg.IsFeatureToggleEnabled(featuremgmt.FlagDatabaseMetrics) {
+		metrics = sqlstats.NewStatsCollector("grafana", db)
+	} else {
+		// TODO: deprecate/remove these metrics
+		metrics = newSQLStoreMetrics(db)
```

Comment:
I vote for removing it now. Probably only grafana cloud using it and maybe a couple more

Reply:
ok - I know @bergquist is also in favour of removing it now...

In that case, should I put the new metric names under the `database_metrics` feature toggle, or expose them by default?

Reply:
> expose them by default?

expose them by default. `database_metrics` is behind a feature flag cause we only tested it with mysql enough to make me comfortable.

Reply:
I guess `database_metrics` could be turned into a setting instead. In another PR

Reply:
> expose them by default. `database_metrics` is behind a feature flag cause we only tested it with mysql enough to make me comfortable.

Didn't we have a serious bug enabling the feature toggle on non-mysql drivers? With migrations

Reply:
Sorry, disregard my earlier comment, I mixed this up with https://github.com/grafana/grafana/blob/fc348e62796db7d748f032c3ef079656e3fb379d/pkg/services/sqlstore/sqlstore.go#L354-L356